### PR TITLE
liblerc 4.0.0 (new formula)

### PR DESCRIPTION
**Pull request by jshrake**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Review by iMichka — DISMISSED** [review]

File: Formula/liblerc.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      }
+    EOS
+
+    system ENV.cxx, "test.cc", "-std=c++11",
+                    "-I#{include}",
+                    "-L#{lib}",
+                    "-lLerc",
+                    "-o", "test_liblerc"
+
+    assert_equal("", shell_output("./test_liblerc"))
```
Comment (chenrui333) [inline Formula/liblerc.rb:30]:
```suggestion
    assert_empty shell_output("./test_liblerc")
```

Comment (chenrui333) [inline Formula/liblerc.rb:30]: Improve the test a bit
```suggestion
    (testpath/"test.cc").write <<~EOS
      #include <Lerc_c_api.h>
      #include <Lerc_types.h>
      int main() {
        const int infoArrSize = (int)LercNS::InfoArrOrder::_last;
        const int dataRangeArrSize = (int)LercNS::DataRangeArrOrder::_last;
        lerc_status hr(0);

        return 0 ;
      }
    EOS

    system ENV.cxx, "test.cc", "-std=gnu++17",
                    "-I#{include}",
                    "-L#{lib}",
                    "-lLerc",
                    "-o", "test_liblerc"

    assert_empty shell_output("./test_liblerc")
```

File: Formula/liblerc.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+  test do
+    (testpath/"test.cc").write <<~EOS
+      #include <Lerc_c_api.h>
+      #include <Lerc_types.h>
+      int main() {
+      }
+    EOS
+
+    system ENV.cxx, "test.cc", "-std=c++11",
```
Comment (chenrui333) [inline Formula/liblerc.rb:24]: note, upstream does support c++17

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
@jshrake, thanks for your contribution to Homebrew! :tada: 🥇

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Pull request closed without merging**